cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -517,3 +517,321 @@ def test_host_comparison_multipleviews_interactions():
             10. Same expected results as for steps 6 through 9.
     """
     pass
+
```

Comment:
I would recommend putting these in a new `test_host_manual.py` file since they're all specific to infra_hosts.

Even these are manual, you can still parametrize them if any should be done for multiple infra provider types.

Reply:
I created test_host_manual.py and moved the hosts tests there.